.changelog/9000.internal.md

```diff
@@ -0,0 +1,18 @@
+fixme: add a creative title
```

Comment:
Could just add an empty trivial file as it fixes something introduced in this release.

Reply:
Yeah I just have that there so I don't get annoyed by ci-lint when I'm still working on a branch.